package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adldap",
-  "version": "3.1.2",
+  "version": "3.1.3",
```

Comment:
Please do not update the version. That is for release maintainers (i.e. me) to do.